main.js

```diff
@@ -1,19 +1,32 @@
 var $carImg = document.querySelector('.car-img');
 
+var direction = {
+  north: {
+    arrow: 'ArrowUp',
+    class: 'carImg up'
+  },
+  south: {
+    arrow: 'ArrowDown',
+    class: 'carImg down'
+  },
+  east: {
+    arrow: 'ArrowRight',
+    class: 'carImg right'
+  },
+  west: {
+    arrow: 'ArrowLeft',
+    class: 'carImg left'
+  }
+};
+
 document.addEventListener('keydown', function (event) {
 
-  if (event.key === 'ArrowRight') {
-    $carImg.className = 'carImg right';
-  }
+  if (event.key === direction.east.arrow) $carImg.className = direction.east.class;
 
-  if (event.key === 'ArrowLeft') {
-    $carImg.className = 'carImg left';
-  }
+  if (event.key === direction.west.arrow) $carImg.className = direction.west.class;
+
+  if (event.key === direction.north.arrow) $carImg.className = direction.north.class;
+
+  if (event.key === direction.south.arrow) $carImg.className = direction.south.class;
 
-  if (event.key === 'ArrowUp') {
-    $carImg.className = 'carImg up';
-  }
-  if (event.key === 'ArrowDown') {
-    $carImg.className = 'carImg down';
-  }
 });
```

Comment:
So this isn't using a data model, this is more of a look up table.  Where you have an object that would translate the event key to the direction.

Here is a small example of a data model being used.
```
var data = {
 direction: 'east'
}
document.addEventListener('keydown', function (event) {
if(event.key === 'ArrowUp') {
  data.direction = 'north'
} //  ...

$carImg.className = data.direction
```

Does that make sense? Your data model should hold data that you will use in your code as a reference. To make sure that you have one source of truth.

So later if you want to know what direction the car is going , you check the code, not the dom